src/models/game.py

```diff
@@ -20,15 +45,56 @@ class Game:
     current_player_id: t.Optional[int]
     step_count: int
     winner_id: t.Optional[int]
-    field: t.Union[str, list]
+    field: t.Union[str, t.List[chr]]
     current_state: str
+    owner_mark: chr = 'X'
+    opponent_mark: chr = '0'
+
+    def _game_loop(self, field: t.List[str]):
+        for condition in WIN_CONDITIONS:
+            if all(field[cell_id] == self._current_mark() for cell_id in condition):
```

Comment:
cell_id kinda confusing here, maby index, or idx, or cell_idx if you wanna mark that is wariable is belongs to cell